### ISSUE TITLE
Welcome to share the paper and code related to 3D medical pre-training

### ISSUE THREAD

**Issue by MrGiovanni** [MrGiovanni/SuPreM#2]
**Suggested format**

- Paper title:
- Author list:
- Paper link:
- Code link:

We have made efforts to include papers related to 3D medical pre-training as comprehensively as possible in [Awesome Medical Pre-Training](https://github.com/MrGiovanni/SuPreM/document/awesome_medical_pretraining.md), but we acknowledge that there may be some important publications that we have missed. Considering the dynamic nature of research in this area, we anticipate that it will gain increasing attention in the near future. Therefore, we are committed to continuously expanding our list over time.

Your valuable input would be highly appreciated in this regard. We welcome any suggestions on publications that we may have overlooked, as well as any new publications that you come across in the future and find relevant to our repository.

**Comment by lorinczszabolcs** [comment]
> Hi! First of all, awesome work!
> 
> Here is a suggestion: Paper title: `MedNeXt: Transformer-Driven Scaling of ConvNets for Medical Image Segmentation` Author list: `Saikat Roy`, `Gregor Koehler`, `Constantin Ulrich`, `Michael Baumgartner`, `Jens Petersen`, `Fabian Isensee`, `Paul F. Jäger` & `Klaus H. Maier-Hein` Paper link: https://link.springer.com/chapter/10.1007/978-3-031-43901-8_39 Code link: https://github.com/MIC-DKFZ/MedNeXt

I just realized that the proposed paper should have gone in the other issue template, to the backbones, should I move it there?

**Comment by Luffy03** [comment]
Thanks for your nice work first! Would you please help us share our CVPR 2024 paper? Many thanks!

Paper title: VoCo: A Simple-yet-Effective Volume Contrastive Learning Framework for 3D Medical Image Analysis
Author list: Linshan Wu, Jiaxin Zhuang, and Hao Chen
Paper link: https://arxiv.org/abs/2402.17300
Code link: https://github.com/Luffy03/VoCo

**Comment by Luffy03** [comment]
Dear Prof. Zhou, many thanks for your advanced works in this field, which inspires us a lot!
Would you please help us share our CVPR 2024 extension paper? Thank you very much!

Paper title: Large-Scale 3D Medical Image Pre-training with Geometric Context Priors
Author list: Linshan Wu, Jiaxin Zhuang, and Hao Chen
Paper link: https://arxiv.org/abs/2410.09890
Code link: https://github.com/Luffy03/Large-Scale-Medical
Dataset link: https://huggingface.co/datasets/Luffy503/PreCT-160K
Dataset link: https://huggingface.co/datasets/Luffy503/VoComni
Models link: https://huggingface.co/Luffy503/VoCo

**Comment by FengheTan9** [comment]
Dear Zhou, we would greatly appreciate it if you could help share our MICCAI 2024 early acceptance and MedIA 2025 paper. Thank you! 🤗🤗🤗

Paper title: HySparK: Hybrid Sparse Masking for Large Scale Medical Image Pre-Training (MICCAI 2024)
Author list: Fenghe Tang, Ronghao Xu, Qingsong Yao, Xueming Fu, Quan Quan, Heqin Zhu, Zaiyi Liu, and S. Kevin Zhou
Paper link: [https://papers.miccai.org/miccai-2024/paper/0897_paper.pdf](https://papers.miccai.org/miccai-2024/paper/0897_paper.pdf)
Code link: [https://github.com/FengheTan9/HySparK](https://github.com/FengheTan9/HySparK)

Paper title: MambaMIM: Pre-training Mamba with State Space Token Interpolation and its Application to Medical Image Segmentation (MedIA 2025)
Author list: Fenghe Tang, Bingkun Nian, Yingtai Lia, Zihang Jiang, Jie Yang, Wei Liu, S Kevin Zhou
Paper link: [https://arxiv.org/pdf/2408.08070](https://arxiv.org/pdf/2408.08070)
Code link: [https://github.com/FengheTan9/MambaMIM](https://github.com/FengheTan9/MambaMIM)
Hugging face: [https://huggingface.co/FengheTan9/MambaMIM](https://huggingface.co/FengheTan9/MambaMIM)
